test/system/src/test/java/io/pravega/test/system/LargeEventTest.java

```diff
@@ -109,7 +109,7 @@ public void largeEventSimpleTest() {
         @Cleanup
         EventStreamWriter<ByteBuffer> writer = clientFactory.createEventWriter(STREAM_NAME,
                                                                                  new ByteBufferSerializer(),
-                                                                                 EventWriterConfig.builder().build());
+                                                                                 EventWriterConfig.builder().enableLargeEvents(true).build());
```

Comment:
Are the events being written larger than `Serializer.MAX_EVENT_SIZE`? The events being written are exactly of `Serializer.MAX_EVENT_SIZE` bytes, which I don't know if this exercises the large even path. Also, we can just try larger events just to see how it works with really large ones (e.g., `10 * Serializer.MAX_EVENT_SIZE`, for instance).